sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/UnnamedSessionReceiver.java

```diff
@@ -101,9 +105,12 @@
                     return;
                 }
 
-                final String token = CoreUtils.isNullOrEmpty(context.getMessage().getLockToken())
-                    ? context.getMessage().getLockToken()
+                final ServiceBusReceivedMessage message = context.getMessage();
+                final String token = !CoreUtils.isNullOrEmpty(message.getLockToken())
+                    ? message.getLockToken()
                     : "";
+
+                logger.info("Received sessionId[{}] messageId[{}]", context.getSessionId(), message.getMessageId());
```

Comment:
This could be excessive in logs, should this be debug message ?

Reply:
Oh yeah. Let me change this tov erbose.